lib/house-arrest/index.js

```diff
@@ -18,10 +19,11 @@ class HouseArrestService extends BaseServiceSocket {
     this._splitter = new LengthBasedSplitter({
       readableStream: socketClient,
       littleEndian: false,
-      maxFrameLength: MAX_FRAME_SIZE,
+      maxFrameLength: getMaxFrameLength(MAX_FRAME_SIZE),
       lengthFieldOffset: 0,
       lengthFieldLength: 4,
       lengthAdjustment: 4,
+      serviceName: 'HouseArrest',
```

Comment:
nit: you could also use `this.constructor.name` to avoid hardcoding service names